src/session7/challenge/Challenge5.java

```diff
@@ -0,0 +1,25 @@
+package session7.challenge;
+
+import java.time.LocalDate;
+
+public class Challenge5 {
+
+    //5. Is Today a Specific Date?
+    //Description: Write a method named isTodaySpecificDate that checks if today's date is 10th December 2019.
+    //Expected Output:
+    //true if today's date is 2019-12-10.
+    //false otherwise.
+
+    public static void main(String[] args) {
+
+        System.out.println("Today`s date is 2019-12-10:" + isTodaySpecificDate());
+        boolean isTodaySpecific = isTodaySpecificDate();
+    }
+
+    public static boolean isTodaySpecificDate() {
```

Comment:
Our method should have a `LocalDate` parameter, so when we call our method, to can input a specific date.